### DISCUSSION TITLE
Allow rewind on Swipers

### DISCUSSION THREAD

**Discussion by Nazrinn** [elementor/elementor#28379] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Since the Swiper version of Elementor has been updated, it would be really nice if there was a rewind feature instead of an infinite loop.

### Describe the solution you'd like.

I'd like to have the option for sliders to rewind to the beginning when accessing beyond the last slide of a slider, especially if infinite looping is not an options (i.e.: when using CSS counters on each slide).

### Describe alternatives you've considered.

I would need to create a slider from scratch if I wanted to access this feature (possible but tedious and defeating the point of having an Elementor slider), or I could find a way to access the slider initialisation in JS, but I'm not sure that's possible given that this is Elementor code.

### Additional context

Here is a working demo: https://swiperjs.com/demos#rewind

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by m1ga** [comment]
https://developers.elementor.com/expose-swiper-in-elementor/
you can set it using JS

**Reply by Nazrinn** [reply]
Thanks for your help!
